src/mdtest.c

```diff
@@ -373,65 +378,65 @@ static void create_file (const char *path, uint64_t itemNum) {
     }
 
     //create files
-    sprintf(curr_item, "%s/file.%s"LLU"", path, mk_name, itemNum);
+    sprintf(curr_item, "%s/file.%s"LLU"", path, o.mk_name, itemNum);
     VERBOSE(3,5,"create_remove_items_helper (non-dirs create): curr_item is '%s'", curr_item);
 
-    if (make_node) {
+    if (o.make_node) {
         int ret;
         VERBOSE(3,5,"create_remove_items_helper : mknod..." );
 
-        ret = backend->mknod (curr_item);
+        ret = o.backend->mknod (curr_item);
         if (ret != 0)
             FAIL("unable to mknode file %s", curr_item);
 
         return;
-    } else if (collective_creates) {
+    } else if (o.collective_creates) {
         VERBOSE(3,5,"create_remove_items_helper (collective): open..." );
 
-        aiori_fh = backend->open (curr_item, IOR_WRONLY | IOR_CREAT, backend_options);
+        aiori_fh = o.backend->open (curr_item, IOR_WRONLY | IOR_CREAT, o.backend_options);
         if (NULL == aiori_fh)
             FAIL("unable to open file %s", curr_item);
 
         /*
          * !collective_creates
          */
     } else {
-        hints.filePerProc = !shared_file;
+        o.hints.filePerProc = ! o.shared_file;
         VERBOSE(3,5,"create_remove_items_helper (non-collective, shared): open..." );
 
-        aiori_fh = backend->create (curr_item, IOR_WRONLY | IOR_CREAT, backend_options);
+        aiori_fh = o.backend->create (curr_item, IOR_WRONLY | IOR_CREAT, o.backend_options);
         if (NULL == aiori_fh)
             FAIL("unable to create file %s", curr_item);
     }
 
-    if (write_bytes > 0) {
+    if (o.write_bytes > 0) {
         VERBOSE(3,5,"create_remove_items_helper: write..." );
 
         /*
          * According to Bill Loewe, writes are only done one time, so they are always at
          * offset 0 (zero).
          */
-        hints.fsyncPerWrite = sync_file;
-        if(write_bytes >= 8){ // set the item number as first element of the buffer to be as much unique as possible
-          ((uint64_t*) write_buffer)[0] = itemNum;
+        o.hints.fsyncPerWrite = o.sync_file;
+        if(o.write_bytes >= 8){ // set the item number as first element of the buffer to be as much unique as possible
```

Comment:
(style) "if (" and ") {"

Reply:
I see where you come from, maybe time to thing about some coding conventions. In terms of what is used in the repo though:
$ grep if src/*.c| cut -d ":" -f 2- |grep "if([a-zA-Z0-9]" | wc -l
264
$ grep "if ( " src/*.c| cut -d ":" -f 2- |grep "if([a-zA-Z0-9]" | wc -l
70
In that sense, the style I used is dominant 80%. I personally also think this isn't that important (right now) but yeah, maybe coding conventions.